lib/pbench/agent/results.py

```diff
@@ -88,11 +88,13 @@ def __init__(
         self.controller = controller
         if (self.result_dir.parent / f"{self.result_dir.name}.copied").exists():
             raise self.AlreadyCopied(f"Already copied {self.result_dir.name}")
-        if (self.result_dir / ".running").exists():
+        running = self.result_dir / ".running"
+        if running.exists():
+            running_str = f"{running.parent.name}/{running.name}"
             raise self.BenchmarkRunning(
-                f"The benchmark is still running in {self.result_dir.name} - skipping; "
-                f"if that is not true, rmdir {self.result_dir.name}/.running, "
-                "and try again",
+                f"The benchmark is still running in {self.result_dir.name} -"
+                f" skipping; if that is not true, rmdir {running_str}, and try"
+                " again",
```

Comment:
Semantically, especially with the line breaks here, it almost looks like the "if that is not true" is attached to the "skipping", and "skipping" is not subject to interpretation in this context. On one line it won't be quite so odd, but I think that we might still be better off _starting_ messages like this with the action and expanding on that with the reasoning behind the action. E.g., `"Skipping {self.result_dir.name}: the benchmark appears to be running. If that's incorrect, delete {running_str} and try again"`. (I'd rather not prescribe `rmdir` -- for example, I always use `rm -r` -- but more importantly, isn't `.running` a _file_ rather than a directory?)